docs/gui/wpf.md

```diff
@@ -9,22 +9,16 @@ At this point, no group in the Mono project has plans to implement Windows Prese
 
 We do not have any plans because the project is too large and there has not been any serious interest from the community to make this effort move forward.
 
-Some bits have been implemented for WindowsBase and they are distributed with Mono (mostly because System.IO.Packaging is part of WindowsBase). Various classes and stubs were developed and live in the [Olive](/archived/olive) module.
+Some bits were implemented for WindowsBase and they are distributed with Mono (mostly because System.IO.Packaging is part of WindowsBase). Various classes and stubs were developed and live in the [Olive](/archived/olive) module.
 
-Anyone interested in contributing should follow the guidelines in the [Contributing](/community/contributing/) page.
-
-At this point, we strongly suggest that users interested in WPF adopt Silverlight instead as it can now be used outside of the browser and offers a rich set of cross platform APIs and features.
+At this point, we strongly suggest that users interested in WPF adopt Xamarin.Forms instead as it offers a rich set of cross platform APIs and features (and has a WPF driver too).
```

Comment:
The audience of this page probably is not interested in Xamarin.Forms WPF backend, but GTK# backend.

However, neither Xamarin.Forms GTK# backend, nor Maui's Linux support is for production (in preview or community driven), so it is rather a abandoned field.

Reply:
I'd say the updated wording is still a lot better than what we had before :)